appveyor.yml

```diff
@@ -6,7 +6,7 @@ clone_folder: c:\gopath\src\go.opencensus.io
 
 environment:
   GOPATH: c:\gopath
-  GOVERSION: 1.8
+  GOVERSION: 1.6
```

Comment:
Is there a way to run both with 1.6 and 1.9?

Reply:
Am not sure, I spent a lot of time trying to research about this, with no results. I'll defer to someone with more knowledge but it def runs on Travis CI.